core/src/com/unciv/logic/GameSaver.kt

```diff
@@ -57,7 +57,11 @@ object GameSaver {
     }
 
     fun getGeneralSettingsFile(): FileHandle {
-        return Gdx.files.local(settingsFileName)
+        try {
+            return Gdx.files.local(settingsFileName)
+        } catch (ex: NullPointerException) {
+            return FileHandle(settingsFileName)
```

Comment:
Why are you having local files replaces with filehandles everywhere? I must be missing something

Reply:
I was thinking to run in console mode without starting libgdx module Gdx.
And for all file IO we are using Gdx.files which is initialized after Gdx.app start.
For full console I was just importing FileHandle which doesn't need Gdx.app running it's just a wrapper for Java IO.
In such way I don't need to run LwjglApplication(game, config) and can make console mode more lightweight and quick.

Probably would be better to change to smth like to make it clear:
```
        if (UncivGame.consoleMode) {
            return FileHandle(settingsFileName)
        } else {
            return Gdx.files.local(settingsFileName)
```